docs/modules/ROOT/pages/type-definitions/schema-configuration/global-configuration.adoc

```diff
@@ -0,0 +1,50 @@
+[[schema-configuration-global-configuration]]
+= Global Configuration
+
+There are cases when what is wanted is to disable a type of operation for the whole type definitions and not on type by type basis.
+For instance, let's say that we want to avoid all the top-level aggregation operations to be generated. To avoid useless repetition, Neo4j GraphQL Library allows using schema extension with `@query`, `@mutation`, and `@subscription` as follows:
```

Comment:
```suggestion
For instance, if you want to avoid all the top-level aggregation operations to be generated and avoid redundancy, Neo4j GraphQL Library allows using schema extensions with `@query`, `@mutation`, and `@subscription`:
```